app/helpers/stories_helper.rb

```diff
@@ -27,4 +27,18 @@ def is_unread?(comment)
 
     (comment.created_at > @ribbon.updated_at) && (comment.user_id != @user.id)
   end
+
+  def self.repost_story_description(story, story_params)
```

Comment:
The helpers in `StoriesHelper` are for the view, please move this up to a private method in the controller.

Reply:
Resolved.